### ISSUE TITLE
Feature Request: Mi Band 3: Allow to sort/order displays

### ISSUE THREAD

**Issue by rixx** [Freeyourgadget/Gadgetbridge#1298]
With my Mi Band 3, Gadgetbridge allows me to select which screens to display (Settings > Mi Band 3 > Display items), but I can't change their order. This is possible in the original Mi app, and since the Gadgetbridge default ordering is very unintuitive for me, I'd prefer to change the order manually.

Gadgetbridge version is: 0.30.0

**Comment by LuccoJ** [comment]
The same applies to the Bip, although I don't know if the protocol messages used to change the order is the same as in the Mi Band 3, and I've been having issues obtaining a Bluetooth log lately.

**Comment by joserebelo** [comment]
Yes, it seems other devices can be reordered, including the Bip, see https://github.com/Freeyourgadget/Gadgetbridge/pull/1202#issuecomment-412304061. The logic is already there, all that's missing is the UI implementation on the AppManager.
